wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/MessageAction.swift

```diff
@@ -16,11 +16,32 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
 import WireCommonComponents
+import WireDataModel
 import UIKit
 
-enum MessageAction: CaseIterable {
+enum MessageAction: CaseIterable, Equatable {
+    static var allCases: [MessageAction] = [.digitallySign,
```

Comment:
Why did you implement a custom `allCases` ?

Reply:
because .react require associated value